package-lock.json

```diff
@@ -18,6 +18,7 @@
         "vscode-extension-telemetry": "^0.1.7",
         "vscode-languageclient": "^7.0.0-next.12",
         "vscode-uri": "^3.0.2",
+        "which": "^2.0.2",
```

Comment:
I'm not sure why does the addition of this single dependency cause such a messy diff 🤷🏻

Reply:
Maybe there were outdated things in the Lockfile (so it was not consistent with the dependencies in package.json anymore). But I don't know 😄